src/py42/clients/alerts.py

```diff
@@ -42,23 +45,44 @@ def get_details(self, alert_ids):
     def resolve(self, alert_ids, reason=None):
         """Resolves the alerts with the given IDs.
 
+        Deprecated. Support will be removed from version 2.0.0. Use `update_state` instead.
+
         Args:
             alert_ids (iter[str]): The identification numbers for the alerts to resolve.
             reason (str, optional): The reason the alerts are now resolved. Defaults to None.
 
         Returns:
             :class:`py42.response.Py42Response`
         """
-        return self._alert_service.resolve(alert_ids, reason=reason)
+        return self._alert_service.update_state(
+            AlertState.DISMISSED, alert_ids, reason=reason
+        )
 
     def reopen(self, alert_ids, reason=None):
         """Reopens the resolved alerts with the given IDs.
 
+        Deprecated. Support will be removed from version 2.0.0. Use `update_state` instead.
+
+        Args:
+            alert_ids (iter[str]): The identification numbers for the alerts to reopen.
+            reason (str, optional): The reason the alerts are reopened. Defaults to None.
+
+        Returns:
+            :class:`py42.response.Py42Response`
+        """
+        return self._alert_service.update_state(
+            AlertState.OPEN, alert_ids, reason=reason
+        )
+
+    def update_state(self, status, alert_ids, reason=None):
```

Comment:
See https://github.com/code42/py42/pull/186/files#r485637405 ....

This does not mean you should change the parameter name in `resolve()` or `reopen()`... Leaving those as `reason` is perfectly fine, especially if it's deprecated.